apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -97,7 +102,11 @@ export abstract class BaseInstallAction extends BaseRushAction {
       StandardScriptUpdater.validate(this.rushConfiguration);
     }
 
-    this.eventHooksManager.handle(Event.preRushInstall, this.parser.isDebug);
+    if (!this._ignoreHooksParameter.value) {
+      this.eventHooksManager.handle(Event.preRushInstall, this.parser.isDebug);
+    } else {
+      console.log(`The --ignore-hooks flag was specified, skipping preInstall hooks`);
```

Comment:
We should probably only log these messages if any hooks are actually defined.

Reply:
Does it make sense to delegate this behavior to the EventHooksManager, then?

Reply:
I think so.

Reply:
Moved.